projects/core/styles/theme/wrapper/base.less

```diff
@@ -1,8 +1,8 @@
 @import '../../taiga-ui-local.less';
 
 /* stylelint-disable order/order */
-tui-wrapper,
-[tuiWrapper] {
+tui-wrapper:not([data-appearance='none']),
+[tuiWrapper]:not([data-appearance='none']) {
```

Comment:
Gotta be careful not to mess the specificity of some rules that override those. Just keep that in mind, screenshot tests might not catch some tricky case

Reply:
You are right.
The increased specificity breaks other appearance.

Luckily, our screenshot tests works))

Reply:
fixed